coreMQTT/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+COREMQTT_VERSION="2.3.0"
+COREMQTT="coreMQTT-${COREMQTT_VERSION}"
+PKG_URL="https://github.com/FreeRTOS/coreMQTT/archive/refs/tags/"
+
+
+PREFIX_PORT_SRC="${PREFIX_PORT_BUILD}/${COREMQTT}/source"
+PREFIX_PORT_STAGING="${PREFIX_PORT_BUILD}/staging"
+
+
+b_port_download "${PKG_URL}" "v${COREMQTT_VERSION}.tar.gz"
```

Comment:
Maybe use
`b_port_download "${PKG_URL}" "${COREMQTT}.tar.gz" "v${COREMQTT_VERSION}.tar.gz"`
instead.

If the third parameter is not passed, then the second parameter is the filename used in the download URL (both in PKG_URL and mirror URL) and as a destination filename.

If the third parameter is passed, then it will be used as a filename for the first download (via PKG_URL) and the downloaded file will be renamed to the filename provided in the second parameter. The second parameter will be used as a filename in mirror URL. (See phoenix-rtos-ports/build.subr)

Why does that matter? If we download something from the internet directly (e.g. from github), the source package may be named e.g. `v2.3.0.tar.gz` or `86a78f67b....tar.gz` (hash). We want to use `packagename-version.ext` notation so that it is clear what was downloaded and it will be easier for files on the mirror to coexist.

(Sorry, I missed that.)

Reply:
Done :)